.github/workflows/cicd_test_trigger.yml

```diff
@@ -22,7 +22,8 @@ jobs:
 
   DownloadAndOutput:
     runs-on: ubuntu-latest
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    #if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
it's better to restore this change, because VS daily release with TTK daily build

Reply:
Thanks for reminding me - I will create a new PR on dev to rollback this.